### ISSUE TITLE
train mode was not turned on by default during training

### ISSUE THREAD

**Issue by wdyin** [NVlabs/SPADE#123]
I notice that model.train() is not called and thus all bn layers of the generator are in eval mode by default during training process. I wonder if it's a bug or the author just implemented it like that.
